### PR TITLE
fix linter issues

### PR DESCRIPTION
**Pull request by jmank88**
This PR fixes some linter issues, and removes some unused code that contained linter issues.

### CODE REVIEW

File: core/chains/evm/gas/block_history_estimator.go [hunk 1]
```diff
@@ -340,9 +340,9 @@ func (b *BlockHistoryEstimator) haltBumping(attempts []EvmPriorAttempt) error {
 	}
 	// Return error to prevent bumping if gas price is nil or if EIP1559 is enabled and tip cap is nil
 	if maxGasPrice == nil || (b.eConfig.EIP1559DynamicFees() && maxTipCap == nil) {
-		errorMsg := fmt.Sprintf("%d percentile price is not set. This is likely because there aren't any valid transactions to estimate from. Preventing bumping until valid price is available to compare", percentile)
-		b.logger.Debugf(errorMsg)
-		return errors.New(errorMsg)
+		err := fmt.Errorf("%d percentile price is not set. This is likely because there aren't any valid transactions to estimate from. Preventing bumping until valid price is available to compare", percentile)
+		b.logger.Debugw("Invalid configuration", "err", err)
```
Comment (amit-momin) [inline core/chains/evm/gas/block_history_estimator.go]: I think "Invalid configuration" does't describe this accurately. Maybe to keep the original behavior couldn't we just do 
```suggestion
		b.logger.Debug(err.Error())
```

File: core/services/pipeline/getters.go [hunk 2]
```diff
@@ -147,7 +147,7 @@ func JSONWithVarExprs(jsExpr string, vars Vars, allowErrors bool) GetterFunc {
 				}
 				keypath, is := maybeKeypath.(string)
 				if !is {
-					return nil, errors.Wrapf(ErrBadInput, fmt.Sprintf("you cannot use %s in your JSON", chainlinkKeyPath))
+					return nil, errors.Wrap(ErrBadInput, fmt.Sprintf("you cannot use %s in your JSON", chainlinkKeyPath))
```
Comment (amit-momin) [inline core/services/pipeline/getters.go:150]: Wouldn't we want to use `fmt.Errorf()` here?
```suggestion
					return nil, fmt.Errorf("you cannot use %s in your JSON: %w", chainlinkKeyPath, ErrBadInput)
```

File: core/chains/evm/gas/block_history_estimator.go (hunk 1 above)
Comment (jmank88) [reply core/chains/evm/gas/block_history_estimator.go]: Ah, I misread the first sentence as referring to configuration. I think it is important to follow our standard for errors in structured logs here. What would be a more accurate message?
```suggestion
		b.logger.Debugw("Bumping halted", "err", err)
```

File: core/services/pipeline/getters.go (hunk 2 above)
Comment (jmank88) [reply core/services/pipeline/getters.go:150]: :shrug: I would love to remove `pkg/errors` everywhere, but there are objections from folks who want stack traces on all errors.

File: core/chains/evm/gas/block_history_estimator.go (hunk 1 above)
Comment (amit-momin) [reply core/chains/evm/gas/block_history_estimator.go]: I think that's better!

File: core/services/pipeline/getters.go (hunk 2 above)
Comment (amit-momin) [reply core/services/pipeline/getters.go:150]: No worries in that case

**Review by amit-momin — APPROVED** [review]

**Review by george-dorin — APPROVED** [review]

**Pull request merged**